source/slides/03_linux_basics.rst

```diff
@@ -0,0 +1,443 @@
+.. _03_linux_basics:
+
+Linux Basics (Day 3)
+====================
+
+Filesystems, system booting and OS Installation
+
+Before we begin
+---------------
+
+.. code-block:: bash
+
+  $ sudo yum install man man-pages man-pages-overrides
+  $ man hier # a massive man page explaining the filesystem hierarchy
+
+.. code-block:: bash
+
+  $ echo $PATH
+  /usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:\
+  /home/centos/bin
+
+``$PATH`` is used as a colon-delimited list of places to look for binaries by
+your shell
+
+.. code-block:: bash
+
+  $ which pwd
+  /bin/pwd
+
+The ``which`` command tells you which binary it is running for a given name.
+
+Special File permissions
+------------------------
+
+* Sticky bit
+* setuid
+* setgid
+* Security
+
+Filesystem
+==========
+
+The Linux Filesystem Hierarchy
+------------------------------
+
+What's a filesystem?
+
+    *In computing, a file system is used to control how information is stored and
+    retrieved. Without a file system, information placed in a storage area would
+    be one large body of information with no way to tell where one piece of
+    information stops and the next begins.*
+
+    -- (http://en.wikipedia.org/wiki/Filesystem)
+
+Filesystem can mean:
```

Comment:
I didn't have anything specific in mind, but we could talk about some more sysadmin-specific file system differences & niceties. Why, from a sysadmins perspective, would you pick $FS.